Lib/urllib/request.py

```diff
@@ -881,15 +881,15 @@ def reduce_uri(self, uri, default_port=True):
         return authority, path
 
     def is_suburi(self, base, test):
-        """Check if test is below base in a URI tree
+        """Check if test is below base in a URI tree (inclusive)
```

Comment:
What does 'inclusive' mean here?

Reply:
e.g, is_suburi(base, base) is True

Reply:
Should i rephrase?

Reply:
If we look at is_suburi as an ordering function, this is weak ordering

Reply:
Would 'test equal or below base' be accurate?  I know what partial order is, not 'weak order'.

Reply:
Yes, i meant partial ordering
```suggestion
        """Check if test is equal or below base in a URI tree
```